internals/overlord/state/notices.go

```diff
@@ -26,6 +28,11 @@ import (
 const (
 	// defaultNoticeExpireAfter is the default expiry time for notices.
 	defaultNoticeExpireAfter = 7 * 24 * time.Hour
+
+	// defaultNoticeRecipientUserID is the UID of the default user with whom
+	// notices will be associated. A UID of -1 means the notice should be
+	// readable by any user.
+	defaultNoticeRecipientUserID = -1
```

Comment:
Seeing we don't use "recipient" elsewhere, how about just:
```suggestion
	defaultNoticeUserID = -1
```